db/schema.rb

```diff
@@ -635,6 +650,97 @@
      FROM (appointments ap
        JOIN facilities f ON ((ap.facility_id = f.id)));
   SQL
+  create_view "latest_blood_pressures_per_patient_per_days", materialized: true, sql_definition: <<-SQL
```

Comment:
This diff should be removed. We really need to figure out how to not make massive diffs for the schema file every time.